config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id/add_room/:room_id", to: "guests#add_room"
+  post "/guests/:id", to: "guests#submit"
 end
```

Comment:
Careful here, these bottom two routes are not restful. The route for line 7 should be `post "/guests/:id/rooms", to: "guest_rooms#create"` controller could be something else, but remember the routes should always be one of the actions: index, show, new, create, edit, update, or destroy and our url paths should always follow RESTful  convention.